epubhv/epubhv.py

```diff
@@ -34,15 +37,18 @@ def list_all_epub_in_dir(path):
     files = []
     for root, _, filenames in os.walk(path):
         for filename in filenames:
-            files.append(os.path.join(root, filename))
+            relative_path = os.path.join(
+                root, filename).replace("\\", "/")  # windows
+            files.append(relative_path)
```

Comment:
sorry for this, I forgot to use `pathlib Path` here, could you help me change it, then we can pass the windows things instead of  replace `\\`

Reply:
I'll fix this later today